blockchain-importer/src/Pos/BlockchainImporter/Tables/PendingTxsTable.hs

```diff
@@ -0,0 +1,89 @@
+import           Data.Time.Clock.POSIX (getCurrentTime)
+import qualified Database.PostgreSQL.Simple as PGS
+import           Opaleye
+
+import           Pos.BlockchainImporter.Tables.TxAddrTable (TxAddrRowPGR, TxAddrRowPGW,
+                                                            addressToTxTable)
+import qualified Pos.BlockchainImporter.Tables.TxAddrTable as TAT (insertTxAddresses)
+import           Pos.BlockchainImporter.Tables.Utils
+import           Pos.Core.Txp (Tx (..), TxOut (..), TxOutAux (..), TxUndo)
+import           Pos.Crypto (hash)
+
+data PTxRowPoly h iAddrs iAmts oAddrs oAmts t = PTxRow  { ptrHash          :: h
+                                                        , ptrInputsAddr    :: iAddrs
+                                                        , ptrInputsAmount  :: iAmts
+                                                        , ptrOutputsAddr   :: oAddrs
+                                                        , ptrOutputsAmount :: oAmts
+                                                        , ptrCreatedTime   :: t
+                                                        } deriving (Show)
+
+type PTxRowPGW = PTxRowPoly (Column PGText)
+                            (Column (PGArray PGText))
+                            (Column (PGArray PGInt8))
+                            (Column (PGArray PGText))
+                            (Column (PGArray PGInt8))
+                            (Column (PGTimestamptz))
+type PTxRowPGR = PTxRowPoly (Column PGText)
+                            (Column (PGArray PGText))
+                            (Column (PGArray PGInt8))
+                            (Column (PGArray PGText))
+                            (Column (PGArray PGInt8))
+                            (Column (PGTimestamptz))
+
+$(makeAdaptorAndInstance "pPTxs" ''PTxRowPoly)
+
+pTxsTable :: Table PTxRowPGW PTxRowPGR
+pTxsTable = Table "pending_txs" (pPTxs PTxRow { ptrHash           = required "hash"
+                                              , ptrInputsAddr     = required "inputs_address"
+                                              , ptrInputsAmount   = required "inputs_amount"
+                                              , ptrOutputsAddr    = required "outputs_address"
+                                              , ptrOutputsAmount  = required "outputs_amount"
+                                              , ptrCreatedTime    = required "created_time"
+                                              })
+
+ptxAddrTable :: Table TxAddrRowPGW TxAddrRowPGR
+ptxAddrTable = addressToTxTable "ptx_addresses"
+
+-- | Inserts a given pending Tx into the pending tx tables.
+insertPendingTx :: Tx -> TxUndo -> PGS.Connection -> IO ()
+insertPendingTx tx txUndo conn = do
+  insertPTxToHistory conn tx txUndo
+  TAT.insertTxAddresses ptxAddrTable conn tx txUndo
+
+-- | Inserts the info of a given pending Tx into the master pending Tx table.
+insertPTxToHistory :: PGS.Connection -> Tx -> TxUndo -> IO ()
```

Comment:
the order that the params are received, change in `insertPendingTx`. please use only one convention.